.github/workflows/ci.yml

```diff
@@ -51,114 +60,21 @@ jobs:
           sudo locale-gen en_US.UTF-8
           sudo update-locale LANG=en_US.UTF-8
 
-      - name: Run pipeline with test data
-        uses: Wandalen/wretry.action@v1.0.11
-        with:
-          command: nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results ${{ matrix.parameters }}
-          attempt_limit: 3
-
-  motus:
-    name: Test mOTUs with workflow parameters
-    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/taxprofiler') }}
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        NXF_VER:
-          - "23.04.0"
-          - "latest-everything"
-
-    steps:
-      - name: Check out pipeline code
-        uses: actions/checkout@v2
-
-      - name: Install Nextflow
-        uses: nf-core/setup-nextflow@v1
-        with:
-          version: "${{ matrix.NXF_VER }}"
-
-      - name: Show current locale
-        run: locale
-
-      - name: Set UTF-8 enabled locale
-        run: |
-          sudo locale-gen en_US.UTF-8
-          sudo update-locale LANG=en_US.UTF-8
-
-      - name: Prepare the database
-        run: |
-          wget https://raw.githubusercontent.com/motu-tool/mOTUs/master/motus/downloadDB.py
-          python downloadDB.py > download_db_log.txt
-          echo 'tool,db_name,db_params,db_path' > 'database_motus.csv'
-          echo 'motus,db_mOTU,,db_mOTU' >> 'database_motus.csv'
-
-      - name: Run pipeline with test data
+      - name: Run pipeline for motus
         uses: Wandalen/wretry.action@v1.0.11
+        if: ${{ matrix.tags == 'test_motus' }}
         with:
-          command: nextflow run ${GITHUB_WORKSPACE} -profile test_motus,docker --outdir ./results --databases ./database_motus.csv
+          command: |
+            wget https://raw.githubusercontent.com/motu-tool/mOTUs/master/motus/downloadDB.py
+            python downloadDB.py > download_db_log.txt
+            echo 'tool,db_name,db_params,db_path' > 'database_motus.csv'
+            echo 'motus,db_mOTU,,db_mOTU' >> 'database_motus.csv'
```

Comment:
And these lines need to be replace with a bash if else statement (if matrix.tags), and merged with the main tests, and the github actions YAML `if:` statement needs to be removed -> allowing to have a single test, entry in the YAML and the conditionals within the `command:` block (I hope that makes sense!)

Reply:
I am not sure about this error. 
<img width="792" alt="image" src="https://github.com/nf-core/taxprofiler/assets/64467552/667804d8-cd40-4d7c-80a6-f0150c1c2828">

Reply:
You'll need to check the bash, something is stopping the full evaluation before the final `fi`. You could try  copy pasting into shellcheck and maybe that will give you more info

Reply:
I agree with you. I though it could be caused by indentation. But the command lines work in shell if I just replace the 2 space (default in yml ) into tab (default in shell). No idea how to solve it.

Reply:
Thanks to @samuell the issue was solved. It was not allowed to use multiline code within a bash block while using   `Wandalen/wretry.action@v1.0.11`